### ISSUE TITLE
[BUG] msgpack.py: 128: DeprecationWarning: encoding is deprecated, Use raw=False instead

### ISSUE THREAD

**Issue by jude** [saltstack/salt#57501]
**Description**
A lot of my minions are logging the message

```
[WARNING ] /usr/lib/python3.6/site-packages/salt/utils/msgpack.py:128: DeprecationWarning: encoding is deprecated, Use raw=False instead.
```

This issue looks related to Issue #51398, but is surfacing at another location

**Setup**
N/A

**Steps to Reproduce the behavior**
Control reaches https://github.com/saltstack/salt/blob/master/salt/serializers/msgpack.py#L49 with salt.utils.msgpack.version >= (0.5.2)

In this case, options.setdefault() will set encoding='utf-8', but encoding is deprecated asmsgpack version 0.5.2. The options.setdefault() call should be setting raw=False in this case.

**Expected behavior**
salt/serializers/msg.py.py doesn't log DeprecationWarnings

**Screenshots**
N/A

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
Salt Version:
           Salt: 3000.3

Dependency Versions:
           cffi: Not Installed
       cherrypy: unknown
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
         Jinja2: 2.8.1
        libgit2: Not Installed
       M2Crypto: 0.35.2
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.2
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.6.8 (default, Mar 30 2020, 17:04:00)
   python-gnupg: Not Installed
         PyYAML: 3.12
          PyZMQ: 15.3.0
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.1.4

System Versions:
           dist: centos 7.8.2003 Core
         locale: UTF-8
        machine: x86_64
        release: 4.4.131-1.el7.centos.x86_64
         system: Linux
        version: CentOS Linux 7.8.2003 Core

```
</details>

**Additional context**

It looks like the fix should look something like the block at https://github.com/saltstack/salt/blob/master/salt/transport/ipc.py#L179

**Comment by cmcmarrow** [comment]
@jude I'm working on this issue right now.  I should have a fix in for this soon. Just some questions for you. Why don’t you use msgpack 1.0.0 you wont get the warning with that version. Also what is your log level set at? Because salt logs all the args it removed from msgpack at the info level.

**Comment by oeuftete** [comment]
> @jude I'm working on this issue right now. I should have a fix in for this soon. Just some questions for you. Why don’t you use msgpack 1.0.0 you wont get the warning with that version. Also what is your log level set at? Because salt logs all the args it removed from msgpack at the info level.

@cmcmarrow Even in Salt 3002, msgpack 0.6.2 is packaged with CentOS 7, as the original poster here is using.  Individual users could upgrade, yes, but I think the point is that deprecation warning will be present out of the box and shouldn't be.

That said, this seems to be fixed as of 3002, maybe by f6d6b9d222?

<details>
<summary>3002 example</summary>

```text
[root@master1-five /]# salt -l warning '*' test.ping
minion1:
    True
master1:
    True
raas:
    True
[root@master1-five /]# salt -V
Salt Version:
           Salt: 3002
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
         Jinja2: 2.11.1
        libgit2: Not Installed
       M2Crypto: 0.35.2
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.2
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: Not Installed
   pycryptodome: 3.9.8
         pygit2: Not Installed
         Python: 3.6.8 (default, Apr  2 2020, 13:34:55)
   python-gnupg: Not Installed
         PyYAML: 3.13
          PyZMQ: 17.0.0
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.1.4
 
System Versions:
           dist: centos 7 Core
         locale: UTF-8
        machine: x86_64
        release: 5.4.0-51-generic
         system: Linux
        version: CentOS Linux 7 Core
```

</details>

**Comment by cmcmarrow** [comment]
This PR looks to be the fix, https://github.com/saltstack/salt/pull/51262, if not please reopen the ticket
